es-vue-base/src/lib-components/EsNavBar.vue

```diff
@@ -100,14 +100,48 @@
                                 <slot name="logo" />
                             </template>
                         </es-nav-bar-top-level-menu>
+                        <div
+                            v-if="showSearch"
+                            class="nav-item d-none d-lg-block pt-100">
+                            <es-button
```

Comment:
This button needs a focus state so users navigating by keyboard can see when they're on it

Reply:
It also needs an `aria-label`